apps/web/src/components/digest-demo-flow/DigestDemoFlowProvider.tsx

```diff
@@ -7,14 +7,35 @@ import { useAuthContext } from '../providers/AuthProvider';
 import { useDebounce } from '../../hooks';
 import { useTemplateFetcher, useUpdateTemplate } from '../../api/hooks';
 
-const DigestDemoFlowContext = React.createContext({
+interface IDigestDemoFlowProviderState {
+  isRunningDigest: boolean;
+  triggerCount: number;
+  digestInterval: number;
+  emailsSentCount: number;
+  hoveredHintId?: string;
+}
+
+interface IDigestDemoFlowContextProps {
```

Comment:
This interface implements IDigestDemoFlowProviderState. Maybe we could set it that way. Unless they will grow very apart.